lib/cib/cib_client.c

```diff
@@ -628,7 +628,7 @@ cib_new_variant(void)
     new_cib->cmds->register_callback = cib_client_register_callback;
     new_cib->cmds->register_callback_full = cib_client_register_callback_full;
 
-    new_cib->cmds->noop = cib_client_noop;
+    new_cib->cmds->noop = cib_client_noop; // Deprecated method
```

Comment:
FYI we should keep ping; all daemons should implement a ping (and hopefully one day we can consolidate that code across daemons)

Reply:
Removing support for `CRM_OP_PING` was never under consideration. The only thing I was considering was deprecating the `ping()` method of `cib_api_operations_t`, since it's unused internally.

I'm fine with keeping it though. Like we discussed for `bump_epoch()` and `erase()`, they're potentially useful parts of a complete API, even if we don't use them as such internally.